SetupDataPkg/Tools/ConfigEditor.py

```diff
@@ -535,6 +535,21 @@ def __init__(self, master=None):
                 messagebox.showerror("LOADING ERROR", "Unsupported file '%s' !" % path)
                 return
 
+        if getattr(sys, "frozen", False) and hasattr(sys, '_MEIPASS'):
+            # The application is frozen, pre-populate collected definition files
+            print("Running bundled ConfigEditor! Load pre-populated definition files.\n")
+
+            bundle_dir = sys._MEIPASS
+
+            # The collected definitions will be put under "ConfDefinitions" folder in the bundle directory
+            for subdir, _, files in os.walk(os.path.join(bundle_dir, "ConfDefinitions")):
+                for file in files:
+                    sub_path = os.path.join(subdir, file)
+                    print (sub_path)

```

Comment:
Is the print intended? or is it left over from debugging?
```suggestion

```

Reply:
Ah, good call, it should not be here. Will update.

Reply:
Removed.